File: coresuite-dotnet-standard-cs/generator/TiffToPdf.cs
using ceTe.DynamicPDF;
using ceTe.DynamicPDF.Imaging;

namespace coresuite_dotnet_standard_cs.generator
{
    public class TiffToPdf
    {
        public static void Run(string outputPdfPath)
        {
            // Create a TiffFile object from the TIFF image
            TiffFile tiffFile = new TiffFile(Util.GetResourcePath("Images/fw9_18.tif"));

            // Create a document object from the file
            Document document = tiffFile.GetDocument();

            // Save the PDF document
            document.Draw(outputPdfPath);
        }
    }
}

File: coresuite-dotnet-core-console-vb/generator/TiffToPdf.vb
Imports ceTe.DynamicPDF
Imports ceTe.DynamicPDF.Imaging

Public Class TiffToPdf

    Public Shared Sub Run(outputPdfPath As String)
        ' Create a TiffFile object from the TIFF image
        Dim tiffFile As TiffFile = New TiffFile(Util.GetResourcePath("Images/fw9_18.tif"))

        ' Create a document object from the file
        Dim document As Document = tiffFile.GetDocument()

        ' Save the PDF document
        document.Draw(outputPdfPath)
    End Sub
End Class

